src/App.tsx

```diff
@@ -8,7 +8,7 @@ import ApplyDeveloper from './components/ApplyDeveloper';
 import ApplyProductDesigner from './components/ApplyProductDesigner';
 import ApplyBrandDesigner from './components/ApplyBrandDesigner';
 import Partner from './components/Partner';
-import Projects from './components/Projects';
+import Projects from './components/containers/Projects';
```

Comment:
@rymaju do we want this directory structure?

Reply:
Changed